BigQuery/15_convert_schema_for_labeling.sql

```diff
@@ -0,0 +1,50 @@
+SELECT 
+subject_id as id,
+row_number() over (partition by subject_id, hadm_id, stay_id order by chart_hour asc) as ICULOS,
+admittime as HospAdmTime,
+0 as Unit1,
+1 as Unit2,
+age as Age,
+gender as Gender,
+platelets as Platelets,
+WBC,
+ptt as PTT,
+hemoglobin as Hgb,
+hematocrit as Hct,
+troponin_I as TroponinI,
+bilirubin_total as Bilirubin_total,
+potassium as Potassium,
+phosphate as Phosphate,
+magnesium as Magnesium,
+lactate as Lactate,
+glucose as Glucose,
+bilirubin_direct as Bilirubin_direct,
+creatinine as Creatinine,
+chloride as Chloride,
+calcium as Calcium,
+0 as Alkalinephos,
+0 as BUN,
+0 as AST,
```

Comment:
Are we imputing these to be 0? My take is that n/a values should be left as such. That way the labeler or consumer can make the imputation choice.

Reply:
makes sense to me

Reply:
As before, can always stop the PR and put in a TODO - it doesn't have to be perfect on the first cut.